tests/components/landisgyr_heat_meter/test_init.py

```diff
@@ -20,3 +25,40 @@ async def test_unload_entry(hass):
     assert "landisgyr_heat_meter" in hass.config.components
```

Comment:
I think we should patch the library when setting up the config entry, to make sure that there's no I/O in the tests. Some entities are set to auto poll which could request a coordinator update. It's more robust if we guard for that even though it may not happen at the moment.

Reply:
Never would have thought of that, great. Thanks.